src/contacts/processors/impl/ProcessorFactoryEditOrganization.java

```diff
@@ -0,0 +1,28 @@
+package contacts.processors.impl;
+
+import contacts.processors.IOrganizationActionProcessor;
+import contacts.processors.IOrganizationEditProcessorFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class ProcessorFactoryEditOrganization implements IOrganizationEditProcessorFactory {
```

Comment:
rename to EditOrganizationProcessorFactory

Reply:
Done